worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -3084,4 +3088,33 @@ public void run(Object value) {
         }
         return false;
     }
+
+    @Override
+    public List<? extends Entity> getEntities() {
+        return world.getEntities();
+    }
+    
+    @Override
+    public List<? extends Entity> getEntities(Region region) {
+        return world.getEntities(region);
+    }
+    
+    @Override
+    public Entity createEntity(Location location, BaseEntity entity){
```

Comment:
Missing a space between `)` and `{`

Reply:
fixed in https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/518/commits/947871281f87c0001239cfe78a642322f2361a8b